source/change-streams.rst

```diff
@@ -41,7 +41,35 @@ Terms
 Resumable Error
 ^^^^^^^^^^^^^^^
 
-Any error encountered which is not a server error, with the exception of server responses with the message “not master” or error code 43 (cursor not found).  An example might be a timeout error, or network error.
+Resumable errors consist of:
+
+- any error encountered which is not a server error (e.g. a timeout error or
+  network error)
+
+- server responses classified as "not master" or "node is recovering" errors
```

Comment:
There is an first stab at a compilation of exact errors we should retry on (part of the upcoming 4.0 changestreams cleanup work) [here](https://docs.google.com/document/d/12co9_3nUCvu1W7l2KSGpdDFX_5j4ie8omj1y339KkGY/edit#heading=h.e80mzp6e6w3q). For brevity I'll copy here:
```
Network errors:
NetworkError
HostUnreachable
HostNotFound
NetworkTimeout
SocketException

NotMaster errors:
NotMaster
NotMasterNoSlaveOk
NotMasterOrSecondary
InterruptedDueToReplStateChange
PrimarySteppedDown

Others:
ExceededTimeLimit
RetryChangeStream
```

Totally not expected for you to know this, but I hope it can help round out this list (and the one below) if possible.

Otherwise, LGTM

Reply:
This is missing CursorNotFound, is that intentional? I commented in the doc as well. I can add ExceededTimeLimit and RetryChangeStream, unless I should wait for this doc to be finalized.

Reply:
Hm yeah it looks like it was accidentally omitted. That list has otherwise been somewhat stable, so I'd like to get an answer from Charlie or Spencer and update it in this commit. We'd kill two birds with one stone, since this work needs to be done anyway.

Reply:
Ok, I updated it in this commit. Spencer recommended asking Charlie to take a look.